### ISSUE TITLE
Add ability to pass additional options to Transport for channel construction.

### ISSUE THREAD

**Issue by emkornfield** [googleapis/python-bigquery-storage#46]
This is mainly useful for forcing additional sub-channels to be
created which can increase throughput for read rows calls.

**Comment by meredithslota** [comment]
We've migrated to the microgenerator now so this is (I believe) unblocked. #47 could be reopened or you could make a new PR.

**Comment by emkornfield** [comment]
Yes, I'll try to make a new one in the coming weeks.

**Comment by emkornfield** [comment]
going to close this out for now as I haven't had time to look at it.
